### PR TITLE
list requirements for new CSR

### LINKED ISSUE #4666: Ticket template for replacing custom certificate
Re: [Custom Certificates on Pantheon's Global CDN](https://pantheon.io/docs/custom-certificates/)

Priority: High

## Issue Description:
Some customers frequently update an existing certificate, and aren't able to quickly get it updated.

## Suggested Resolution

Include a ticket template:
https://gist.github.com/ari-gold/5161a8b79b351b561e170fd345158925

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #4666 :metal:

## Effect
PR includes the following changes:
- Lists requirements to request replacement CSR.
-

## Remaining Work
- [x] Technical review (@ari-gold, @ttrowell)
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/_docs/custom-certificates.md [hunk 1]
```diff
@@ -108,6 +108,13 @@ Please allow two business days to get a CSR and load the certificate.
 
 Before your custom certificate expires, Pantheon will open a ticket with your team with a new CSR. You can send that CSR to the Certificate Authority to generate new certificates (as described above for <a href="#option-2-manually-managed-custom-certificates" data-proofer-ignore>bringing a custom certificate</a>).
 
+To request a new CSR ahead of time, please [contact support](/docs/support/) with the following details:
```
Comment (ari-gold) [inline source/_docs/custom-certificates.md]: Thanks @alexfornuto. cc @ttrowell 
How about making this a section that is linkable? Maybe in FAQ?

*What is the process to replace an existing certificate?*

We will provide you with a CSR 45 days prior to your certificate expiration so you can obtain a new certificate. If your use case requires frequently replacing a certificate well ahead of expiration to, for example, add additional domains, then you may replace an existing certificate without needing a new CSR from Pantheon.

To replace a certificate well in advance of expiration, please contact support.....

Comment (alexfornuto) [reply source/_docs/custom-certificates.md]: @ari-gold This is an addendum to an existing FAQ section named "How do I renew my custom certificate". Do you think it needs its own header, or do you agree that it's a good edition to the existing section?

Comment (ari-gold) [reply source/_docs/custom-certificates.md]: @alexfornuto Good call. Yes. looks like it could be combined to that existing FAQ.

**Comment by ttrowell** [comment]
We're getting there!
Along with 

> To update a certificate with additional domains, [contact support](/docs/support/) with the following details:
> 
>  - The current certificate name (CN)
>  - A colon-separated list of domains the certificate is valid for.

We also actually need the updated cert.

**Pull request merged**